cmd/gapit/replace_resource.go

```diff
@@ -50,13 +54,13 @@ func (verb *replaceResourceVerb) Run(ctx context.Context, flags flag.FlagSet) er
 		return nil
 	}
 
-	if verb.Handle == "" {
-		app.Usage(ctx, "-handle argument is required")
+	if (verb.Handle != "" && verb.UpdateResourceBinary != "") || (verb.Handle == "" && verb.UpdateResourceBinary == "") {
```

Comment:
Which is equivalent to:
`if (verb.Handle == "") != (verb.UpdateResourceBinary == "")`

Can keep as-is if you find this clearer.

Reply:
Sorry misread your original suggestion.
Updated, though I added : (verb.Handle == "") == (verb.UpdateResourceBinary == "")
since if condition should be true for error cases

Reply:
Yup, just testing 😉